src/stores/store.jsx

```diff
@@ -2581,29 +2581,16 @@ class Store {
     }
 
     try {
-      const vaultContract = new web3.eth.Contract(asset.vaultContractABI, asset.vaultContractAddress)
-      const controllerAddress = await vaultContract.methods.controller().call({ from: account.address })
-      const controllerContract = new web3.eth.Contract(config.vaultControllerABI, controllerAddress)
+      const url = config.statsProvider + 'vaults'
+      const vaultsApiResultString = await rp(url);
+      const vaults = JSON.parse(vaultsApiResultString)
+      const vault = vaults.find((vault) => vault.address.toLowerCase() === asset.vaultContractAddress.toLowerCase())
 
-      let strategyAddress = ''
-      if(['LINK', 'aLINK'].includes(asset.id)) {
-        strategyAddress = await controllerContract.methods.strategies(asset.vaultContractAddress).call({ from: account.address })
-      } else {
-
-        if(asset.erc20address === 'Ethereum') {
-          asset.erc20address = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
-        }
```

Comment:
One thing to note here is that the vaults API does not currently support ethereum. It supports WETH however. May want to keep this check in for this use case, or add ethereum to vaults API. However, this does not seem to be relevant right now since the ethereum vault doesn't even show up on the statistics page.

Reply:
Good to know. It doesn't seem like it matters for rendering this stat dashboard like you said and since the WETH and ETH vaults are one in the same (at least I think), it makes sense to me that only one shows up in the API. 

What I want to check before this comes in, is if the the drop-down for the ETH vault on the vaults page still renders all the info correctly. If not, then I can add the edge case back in

Reply:
The drop-down still renders correctly with all the strat details